ui/src/components/LandingPage.js

```diff
@@ -12,6 +14,8 @@ class LandingPage extends React.Component {
     return (
       <>
         <Header />
+        <Title className="title" size="4xl">Mobile Apps</Title>
```

Comment:
```suggestion
        <Title className="title" size="2xl">Mobile Apps</Title>

```

Reply:
`4x1` seems larger than Title font in flat

Reply:
I tried out the 2xl. Went with 3xl in the end as it seemed to look the nicer of the 3 options.